src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
```

Comment:
You can apply this rule for all elements with `*` selector instead of add it for card and button